src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculate(double firstOperand, double secondOperand, char operation) {
+        double result;
+        switch (operation) {
+            case '+':
+                result = firstOperand + secondOperand;
+                break;
```

Comment:
Можна одразу повертати значення
```suggestion
                return firstOperand + secondOperand;
              
```